tests/ci/submitpytest.py

```diff
@@ -0,0 +1,177 @@
+ ​
+Microsoft Corporation (“Microsoft”) grants you a nonexclusive, perpetual,
+royalty-free right to use, copy, and modify the software code provided by us
+("Software Code"). You may not sublicense the Software Code or any use of it
+(except to your affiliates and to vendors to perform work on your behalf)
+through distribution, network access, service agreement, lease, rental, or
+otherwise. This license does not purport to express any claim of ownership over
+data you may have shared with Microsoft in the creation of the Software Code.
+Unless applicable law gives you more rights, Microsoft reserves all other
+rights not expressly granted herein, whether by implication, estoppel or
+otherwise. ​
+ ​
+THE SOFTWARE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+MICROSOFT OR ITS LICENSORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THE SOFTWARE CODE, EVEN IF ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGE.
+"""
+
+import os
+import json
+# uncomment if needed
+# import sys
+import azureml.core
+from azureml.core.authentication import AzureCliAuthentication
+from azureml.core import Workspace
+from azureml.core import Experiment
+from azureml.core.runconfig import RunConfiguration
+from azureml.core.conda_dependencies import CondaDependencies
+from azureml.core.runconfig import DEFAULT_CPU_IMAGE
+# uncomment if using gpu
+# from azureml.core.runconfig import DEFAULT_GPU_IMAGE
+from azureml.core.script_run_config import ScriptRunConfig
+
+# this is from
+# https://aidemos.visualstudio.com/DevOps%20for%20AI%20-%20Demo/_git/DevOps-For-AI-AML-Demo?path=%2Faml_service%2F00-WorkSpace.py&version=GBmaster
+# another good ref is
+# https://github.com/Microsoft/MLAKSDeployAML/blob/master/00_AML_Configuration.ipynb
+#
+# Initialize a Workspace
+
+print("SDK Version:", azureml.core.VERSION)
+print('current dir is ' + os.curdir)
+with open("tests/ci/config.json") as f:
+    config = json.load(f)
+
+workspace_name = config["workspace_name"]
+resource_group = config["resource_group"]
+subscription_id = config["subscription_id"]
+location = config["location"]
+
+print(" WS name ", workspace_name)
+print("subscription_id ", subscription_id)
+
+cli_auth = AzureCliAuthentication()
+
+try:
+    print("Trying to get ws")
+    ws = Workspace.get(
+        name=workspace_name,
+        subscription_id=subscription_id,
+        resource_group=resource_group,
+        auth=cli_auth,
+    )
+
+except Exception:
```

Comment:
do we expect the workspace to be deleted? or is this just precautionary?

Reply:
just precautionary. I copied it from the sample code.